services/firebase_cloud_client/firebase_config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "cruizefirestore",
+  "private_key_id": "1d39f091c7b4f30597e7f0b573739bade63b7b0a",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQCr4aq+z8yPJZxA\nSv74UBB0/bpkd36zgnR0bgQ9zvb2YfWDLYGfqf1kdZij4h8zHxE73lA/PclG7a5p\nFk9Bkg5LLM0XFUKA0JaYTKLbUm7vbrbrVp8OQAqCbJ1yOSE+1nwNsAgcr7Meq6FW\nzVn4hY2UB5JN3GXYWv9YTpI+GZ8STJbM8SUU5VDvqiXyLCpbZFHEHvt2z6Gszo1Y\nGmoAnGFVNg+0ghu3XGc7ox8UXnVpwUtGyMeKW7e8sYBBL7FF6XZ6uAPPpv9x7Nwi\nzgAAVSFOa+JuHMFnx83bPrbcAaggHhQPq6qV8E8Vx7GmLxnslDgLhYyLNJIq1pkN\nV0rNFyhDAgMBAAECggEAUc60PmJYCc/MWTt4tIYi9xPmmbfYOac/N0U/xo1RtP31\n5SwNFInLfXuNHX9b+zBvuu96gUcunBA3YXQJX/ucywH1UWXAg+ADtVRQdRP8kgA/\nlq5hQN6RQfdhTsLLRIXAgKFGkyS7G79zqIVPq74hHxIle+KJJNtWyliHOYBnWkUL\nGIx+m7ny5D9XwICjTue58qAKf8ladI0LGb8utVeXPQSjNjLb9tV0X6pHMRygsqv5\nHrm4Ip7HRh3b50Oo3CYvHueh6UwXFI7eZI+5Sv+zOWrW5Xz1U1kra67UrZtJfjDK\nbymheqiAoeNCqv1uuPSDSYcR10cmqhNr/sSWrNtAqQKBgQDS3YARkq6VOdHAXj9e\nhe0EJnVLQ5ehwRRlD/HCsDQSotofB4T/14aRro/J6MPqzLsTQXcrNGGFvuweCOmo\nRvBRtexhcoq0Rd58gkyqVRh6ugw+MMU0MdWTOXb2VNi2KsmWo8O5WQd3e2HTV5NK\nNl9APMd2nAlykDEEIkaRj7b9iQKBgQDQrAcgFpwq/i9Tj3cf1lgKGOaYpJv49kPd\n/i+Vt8X/v2ORFscUiBG4PhbRWlYdxswKV6/DgWwdZG/JnKNAGI75bzv9r0i7ZAM4\ncmSFTROv7jyaVcaWReBQmGn2YzipMs2ReddVh8qRY17Ez0VB7k8IIZUCnbFJEIMr\nM3STZKWwawKBgAieo1iUjWxuM6PQdnivstOjmtHbznv/P40uDKzvtQE6BP6VuRsY\nPVkBMMGJMVEM02zjFTL/7iigeFh19s85LyO38xS8sOzTmleun+GqntiNeGimZ7lN\nfDSLedByAXXY4+qzgRG3kPWSZppUDyJOfLBMi77OfBhhyo8ns4jJvio5AoGAAT+G\nK2L3PCqZDd7lz4Lws97CQld5t1uBvGowzqxyurziU5PXTQAjWv7EFCx+J19xCP7N\nRlYX13UqVgJh15hOAp/vs+zAep52hnfosgNsArliE2o/7dYtKU5AFAEz7Y8oO56G\n8zQEuinl/zdeKq7KOjo5MfLPhKhcoH2GTPtlIlECgYBzb5Y6HNo/04EL+Q+XgjAf\nKbFphiFNQQaITO7f3h3x0aRYb0m1Af4gs3PaEbjSFga9taV5vx6AbMqEtlSR5aRS\nThwWopN8MYHJZoIFK9UH4ygufI0a8fwVHYgKXuTo+2fy/rjhk3+54YOuI44cOvBu\nS5WHVISlQOqLa5m04HRieQ==\n-----END PRIVATE KEY-----\n",
+  "client_email": "firebase-adminsdk-czlva@cruizefirestore.iam.gserviceaccount.com",
+  "client_id": "112727642713260694071",
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://oauth2.googleapis.com/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/firebase-adminsdk-czlva%40cruizefirestore.iam.gserviceaccount.com"
+}
```

Comment:
For now, we'll have our creds here but we'll set up vault hashicorp and that's where these creds will go.